### PR TITLE
1670 rename

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@PavelBal  ALso renamed  f_exp_vasc_endo to f_exp_endosomes because I think this is what it is

**Comment by PavelBal** [comment]
@msevestre In case you didn't see this - I made some comments.

**Comment by msevestre** [comment]
@PavelBal Where?

**Comment by PavelBal** [comment]
image

You can't see these? GitHub was having some issues as I reviewed.

**Comment by msevestre** [comment]
you need to click on the submit review

**Comment by PavelBal** [comment]
Good point :D

File: src/Db/Dump/PKSimDB_dump.txt [hunk 1]
```diff
@@ -4473,13 +4473,13 @@ INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','InitialC
...
-INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','InitialConcentrationEndosome','RC * rel_exp_vasc * f_exp_vasc_endo * 1 / f_endo','Concentration (molar)');
+INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','InitialConcentrationPlasma','V_org > 0 ? RC * (rel_exp_pls + rel_exp_bc * f_exp_bc_membrane * HCT / (1 - HCT) + rel_exp_vasend * f_exp_vasend_plasma * V_vasend / (f_vas * (1 - HCT) * V_org)) : 0','Concentration (molar)');
+INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','InitialConcentrationEndosome','RC * rel_exp_vasend * f_exp_endosomes * 1 / f_endo','Concentration (molar)');
 INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','PARAM_f_exp_bc_membrane','1 - f_exp_bc_cell','Fraction');
-INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','PARAM_f_exp_vasc_tissue_side','1 - f_exp_vasc_endo - f_exp_vasc_apical','Fraction');
+INSERT INTO tab_calculation_method_rates VALUES('ExpressionParameters','PARAM_f_exp_vasc_tissue_side','1 - f_exp_endosomes - f_exp_vasend_plasma','Fraction');
```
[the remark this answers is not in the document]
Comment (msevestre) [reply src/Db/Dump/PKSimDB_dump.txt:4482]: done

File: src/Db/Dump/PKSimDB_dump.txt [hunk 2]
```diff
@@ -88164,26 +88164,26 @@ INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','InitialCo
...
+INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','InitialConcentrationEndosome',240,'Fraction expressed in endosomes','f_exp_endosomes');
 INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','PARAM_f_exp_bc_membrane',240,'Fraction expressed in blood cells','f_exp_bc_cell');
-INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','PARAM_f_exp_vasc_tissue_side',240,'Fraction expressed on plasma-side membrane of vascular endothelium','f_exp_vasc_apical');
-INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','PARAM_f_exp_vasc_tissue_side',240,'Fraction expressed in endosomes','f_exp_vasc_endo');
+INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','PARAM_f_exp_vasc_tissue_side',240,'Fraction expressed on plasma-side membrane of vascular endothelium','f_exp_vasend_plasma');
+INSERT INTO tab_rate_generic_parameters VALUES('ExpressionParameters','PARAM_f_exp_vasc_tissue_side',240,'Fraction expressed in endosomes','f_exp_endosomes');
```
[the remark this answers is not in the document]
Comment (msevestre) [reply src/Db/Dump/PKSimDB_dump.txt:88186]: done

**Comment by msevestre** [comment]
@Yuri05 Also merging the scaling (1-f_vas)= > 1

**Review by Yuri05 — COMMENTED** [review]
looks good

**Pull request merged**